test/cases/0205-tdeepequals-other-functions.lua

```diff
@@ -26,10 +27,21 @@ local test = make_suite("tdeepequals-other-functions")
 
 --------------------------------------------------------------------------------
 
+test '1' (function()
```

Comment:
Test name lacks self-descriptiveness and we should link github issue which describes solved problem tested.